### PR TITLE
Add fdo_sys module documentation

### PR DESCRIPTION
**Pull request by trbehera**
Signed-off-by: Behera, Tushar Ranjan <tushar.ranjan.behera@intel.com>

### CODE REVIEW

File: docs/fdo/fdo-serviceinfo-sys.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+This section defines the “system” module (fdo_sys) specification which provides basic onboarding services
+FDO capable devices. A module is a set of key-value pairs that define the onboarding operations
```
Comment (johneasterday) [inline docs/fdo/fdo-serviceinfo-sys.md]: FDO -> for FDO

File: docs/fdo/fdo-serviceinfo-sys.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+that can be performed on given FDO device. Module key-value pairs are exchanged between the device
```
Comment (johneasterday) [inline docs/fdo/fdo-serviceinfo-sys.md]: on given -> on a given

File: docs/fdo/fdo-serviceinfo-sys.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+and it’s owning Device Management Service. It is up to the owning management service and the
+device to interpret the key-value pairs in accordance to the module specification.
```
Comment (johneasterday) [inline docs/fdo/fdo-serviceinfo-sys.md]: accordance to -> accordance with

File: docs/fdo/fdo-serviceinfo-sys.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+| JSON (readable description) | ["/bin/sh","startup.sh"] |
+| CBOR | `82` # array(2) <br> `67` # text(7) <br> `2F62696E2F7368` # "/bin/sh" <br> `6A` # text(10) <br> `737461727475702E7368` # "startup.sh"|
+| C example | execvp(“/bin/sh”,(char* []) {“shartup.sh”}) |
+
+### fdo_sys:filedesc message
+
+The fdo_sys:filedesc command describes the path to a file the will be used as a part of the
+on-boarding process. A zero-length file is expected to exist on the local file system after this
+command is received. If the described file already exists it is truncated to zero length, otherwise
+a zero-length file is created. The file needs to be created its permissions are set to read/write
```
Comment (johneasterday) [inline docs/fdo/fdo-serviceinfo-sys.md]: created -> created and
permissions are set -> permissions set

File: docs/fdo/fdo-serviceinfo-sys.md (hunk 1 above)
Comment (trbehera) [reply docs/fdo/fdo-serviceinfo-sys.md]: Updated in 06f5e00

File: docs/fdo/fdo-serviceinfo-sys.md (hunk 2 above)
Comment (trbehera) [reply docs/fdo/fdo-serviceinfo-sys.md]: Updated in 06f5e00

File: docs/fdo/fdo-serviceinfo-sys.md (hunk 3 above)
Comment (trbehera) [reply docs/fdo/fdo-serviceinfo-sys.md]: Updated in 06f5e00

File: docs/fdo/fdo-serviceinfo-sys.md (hunk 4 above)
Comment (trbehera) [reply docs/fdo/fdo-serviceinfo-sys.md]: Updated in 06f5e00

**Review by v-vasavi — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Pull request merged**